Tests/TensorFlowTests/TensorAutoDiffTests.swift

```diff
@@ -153,18 +153,17 @@ final class TensorAutoDiffTests: XCTestCase {
         XCTAssertEqual(varianceGradAlongAxes(input), expected)
     }
 
-    // TODO: Uncomment once TF-653 is resolved.
-    // func testTensorInitStacking() {
-    //     let a1 = Tensor<Float>([1, 2, 3, 4, 5])
-    //     let b1 = Tensor<Float>([6, 7, 8, 9, 10])
-    //     let a2 = Tensor<Float>([1, 1, 1, 1, 1])
-    //     let b2 = Tensor<Float>([1, 1, 1, 1, 1])
-    //     let grads = gradient(at: a2, b2) { a, b in
-    //         Tensor<Float>(stacking: [a1 * a, b1 * b], alongAxis: -1).sum()
-    //     }
-    //     XCTAssertEqual(a1, grads.0)
-    //     XCTAssertEqual(b1, grads.1)
-    // }
+    func testTensorInitStacking() {
+        let a1 = Tensor<Float>([1, 2, 3, 4, 5])
+        let b1 = Tensor<Float>([6, 7, 8, 9, 10])
+        let a2 = Tensor<Float>([1, 1, 1, 1, 1])
+        let b2 = Tensor<Float>([1, 1, 1, 1, 1])
+        let grads = gradient(at: a2, b2) { a, b in
+            Tensor<Float>(stacking: [a1 * a, b1 * b], alongAxis: -1).sum()
```

Comment:
Are you still getting a warning?

Reply:
I haven't been able to compile the toolchain from master since the merge that happened today so I couldn't check. I get failures when compiling LLVM but I also recently updated to Catalina so I'm not sure if that's the issue yet.

Reply:
This seems to be passing the CI server so should we go ahead and merge it?

Reply:
yes